includes/Modules/Analytics.php

```diff
@@ -1157,6 +1157,10 @@ private function print_tracking_opt_out() {
 		$account_id  = $settings['accountID'];
 		$property_id = $settings['propertyID'];
 
+		if ( empty( $property_id ) ) {
```

Comment:
Let's check this first before we call `is_tracking_disabled` above as there's no point in invoking all of that if there is no property ID 👍

Reply:
Updated as suggested.